src/libraries/WorkerMessenger.ts

```diff
@@ -47,14 +47,13 @@ export class WorkerMessengerReplyBuffer {
       onceListenerOnly: onceListenerOnly
     };
 
-    if (this.findListenersForMessage(command).length > 0) {
+    if (this.findListenersForMessage(command).length > 0)
       this.replies[command.toString()].push(record);
-    } else {
+    else
       this.replies[command.toString()] = [record];
-    }
   }
 
-  public findListenersForMessage(command: WorkerMessengerCommand): any {
+  public findListenersForMessage(command: WorkerMessengerCommand): any[] {
```

Comment:
should be able to update this return type to `object[]`. not much but it is slightly more restrictive then `any[]` and matches the type definition of `replies`

Reply:
There was a `WorkerMessengerReplyBufferRecord` type already but wasn't being used. Just switched to using this here